src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_input.scss

```diff
@@ -0,0 +1,7 @@
+.workspacePermissionSettingUserAndGroupSelect {
```

Comment:
Would you mind adding some comment on the style overwrite here? I thought OUI component should have the expected styles already.

Reply:
oui only add background for string prepend. For customize component, it will leave an empty background. See below image:
<img width="473" alt="image" src="https://github.com/user-attachments/assets/2c1eec79-292c-476f-8c1d-c6b8edba5897">
@Kapian1234 I think we can add a comment here to describe this case.

Reply:
ok, it's necessary to add a description

Reply:
I kinda feel we should not customize the style of the component from component library, if empty background is what the component library defines, then we should leave as it is, just like the examples on OUI documentation site.